client/dashboard/sites/sync/sync-dropdown.tsx

```diff
@@ -0,0 +1,57 @@
+import { Dropdown, Button, MenuGroup, MenuItem } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { chevronDownSmall, cloudDownload, cloudUpload } from '@wordpress/icons';
+
+interface SyncDropdownProps {
+	className?: string;
+	environment?: 'production' | 'staging';
+}
+
+export default function SyncDropdown( { className, environment }: SyncDropdownProps ) {
```

Comment:
Thanks for adding this in the v2 codebase!

We try to keep the directory hierarchy flat, with common prefix to indicate related areas.

Could we rename this to `client/dashboard/sites/staging-site-sync-dropdown/index.tsx`?